graphql/project_resolver.go

```diff
@@ -42,7 +42,7 @@ func (r *projectResolver) Patches(ctx context.Context, obj *restModel.APIProject
 	apiPatches := []*restModel.APIPatch{}
 	for _, p := range patches {
 		apiPatch := restModel.APIPatch{}
-		err = apiPatch.BuildFromService(p)
+		err = apiPatch.BuildFromService(p, false)
```

Comment:
Right now the project patches query being fetched on spruce fetches the project identifier. Someone from UI may be able to say if it can be removed or not.

Reply:
[the query](https://github.com/evergreen-ci/spruce/blob/main/src/gql/queries/project-patches.graphql#L8)
[the fragment](https://github.com/evergreen-ci/spruce/blob/main/src/gql/fragments/patchesPage.graphql#L6)